bots/notify/src/main/java/org/openjdk/skara/bots/notify/PullRequestWorkItem.java

```diff
@@ -130,31 +176,32 @@ public void run(Path scratchPath) {
                 .materialize(historyPath);
 
         var issues = parseIssues();
-        var prIssues = new PullRequestState(pr, issues);
-        var current = storage.current();
-        if (current.contains(prIssues)) {
+        var commit = resultingCommitHashFor(pr, pr.repository().forge().currentUser());
```

Comment:
The notifier bot may not be running with the same credentials as the commit bot, so this is possibly the wrong user to look for. The other bots (mlbridge, pr) takes a username from the configuration to use for matching comment poster ids.